x/datadeal/keeper/grpc_query_deal_test.go

```diff
@@ -64,86 +64,94 @@ func (suite *queryDealTestSuite) TestQueryDeals() {
 }
 
 func (suite *queryDealTestSuite) TestQueryCert() {
-	cert := &types.Certificate{
-		UnsignedCertificate: &types.UnsignedCertificate{
-			Cid:             "cid1",
-			OracleAddress:   suite.oracleAccAddr.String(),
-			DealId:          1,
-			ProviderAddress: suite.providerAccAddr.String(),
-			DataHash:        "dataHash",
+	consent := &types.Consent{
+		Certificate: &types.Certificate{
+			UnsignedCertificate: &types.UnsignedCertificate{
+				Cid:             "cid1",
+				OracleAddress:   suite.oracleAccAddr.String(),
+				DealId:          1,
+				ProviderAddress: suite.providerAccAddr.String(),
+				DataHash:        "dataHash",
+			},
+			Signature: []byte("signature"),
 		},
-		Signature: []byte("signature"),
 	}
 
-	err := suite.DataDealKeeper.SetCertificate(suite.Ctx, cert)
+	err := suite.DataDealKeeper.SetConsent(suite.Ctx, consent)
 	suite.Require().NoError(err)
 
-	req := &types.QueryCertificate{
-		DealId:   cert.UnsignedCertificate.DealId,
-		DataHash: cert.UnsignedCertificate.DataHash,
+	req := &types.QueryConsent{
+		DealId:   consent.Certificate.UnsignedCertificate.DealId,
+		DataHash: consent.Certificate.UnsignedCertificate.DataHash,
 	}
-	res, err := suite.DataDealKeeper.Certificate(sdk.WrapSDKContext(suite.Ctx), req)
+	res, err := suite.DataDealKeeper.Consent(sdk.WrapSDKContext(suite.Ctx), req)
 	suite.Require().NoError(err)
 	suite.Require().NotNil(res)
-	suite.Require().Equal(cert, res.Certificate)
+	suite.Require().Equal(consent, res.Consent)
 }
 
 func (suite *queryDealTestSuite) TestQueryCerts() {
```

Comment:
```suggestion
func (suite *queryDealTestSuite) TestQueryConsents() {
```

Reply:
Fixed in [bdffe6c](https://github.com/medibloc/panacea-core/pull/590/commits/bdffe6c49e8bdcf116033400a534e5049ef1c3d3).